test/solvers_test.cpp

```diff
@@ -199,6 +199,7 @@ TEST_CASE("MBPT Solver") {
   }
 
   SECTION("Input Data Version") {
+    SKIP("Input version test is disabled until all integral data and throw statement is updated");
```

Comment:
Exactly because of that, I propose to keep `mbpt_outdated_input` to do `input.h5` version testing. And introduce separate version testing for integrals in `green-integrals`. To keep thing close to what they are needed for, importing exceptions from other repository if they do something close to what we need, but not exactly what we need can cause confusions, and also it breaks locality of tests.